### ISSUE TITLE
Standup data in JSON format

### ISSUE THREAD

**Issue by dAnjou** [rouanw/fun-standups#2]
Hey, fun project!

I was wondering, whether the standup data could be converted into JSON format in order to be able to use it API-style from other tools, like CLI or GUI.

**Comment by rouanw** [comment]
Hi @dAnjou. Thanks for your issue; sorry I've only just seen it!

Sounds like a good idea. At the moment it's just deployed with GitHub pages, so there's no API available. That said, by changing the current [`standup-data.js`](https://github.com/rouanw/fun-standups/blob/9c84e90de1a0dec6042eb619af82f88ab5710f37/src/standup-data.js) file to JSON I'm sure we can make some plain JSON available, even if just via GitHub.

I'll take a look.

**Comment by rouanw** [comment]
I've done something simple to start.

@dAnjou, will you please run:

```sh
curl https://raw.githubusercontent.com/rouanw/fun-standups/master/src/standup-data.json -H "Accept: application/json"
``` 

and let me know whether that will work for what you have in mind?

**Comment by dAnjou** [comment]
Hey @rouanw, thanks for doing this! Sorry, I only got to this now.

My motivation came from a little thing I created for my team: https://gitlab.com/dAnjou/bitbar-starter-for-python. We're all using Macs, so I created a little suite of tools to keep track of various things. And part of it is also a "standup helper" which guides us through the standup (it's really just a Google Drawing that is shown in the menu).

And now, thanks to you, I've also included fun standup ideas:
<img width="1029" alt="Screenshot 2020-10-04 at 17 35 32" src="https://user-images.githubusercontent.com/117147/95020510-f8c72480-066b-11eb-9403-2149f569cb46.png">
